### PR TITLE
Add support for disabled option in tw-select. Build.

### PR DESCRIPTION
**Pull request by StevePole**

### CODE REVIEW

File: angular/components/tw-select/tw-select.directive.js [hunk 1]
```diff
@@ -178,7 +181,14 @@
 		function buttonFocus() {
 			$element.triggerHandler('focus');
 		}
-		function optionClick(option) {
+		function optionClick(option, $event) {
+			if (option.disabled) {
+				if ($event.stopPropagation) {
+					$event.stopPropagation();
+				}
+				$event.cancelBubble = true;
```
Comment (oliverviljamaa) [inline angular/components/tw-select/tw-select.directive.js]: Why do we need this in addition to `stopPropagation`?

**Comment by oliverviljamaa** [comment]
Noticed some weird behaviour with `tab` click – it doesn't skip the disabled one and as there's no visible focus state, it seems to just not do anything.

File: angular/components/tw-select/tw-select.directive.js (hunk 1 above)
Comment (StevePole) [inline angular/components/tw-select/tw-select.directive.js]: As far as I'm aware, stopProgogation doesn't have universal browser support.

Comment (oliverviljamaa) [inline angular/components/tw-select/tw-select.directive.js]: Seems to work everywhere except IE < 9:
https://developer.mozilla.org/en/docs/Web/API/Event/stopPropagation
`cancelBubble` is deprecated according to MDN:
https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/cancelBubble

Comment (StevePole) [inline angular/components/tw-select/tw-select.directive.js]: Ok, I'll simplify.

**Pull request merged**